src/channeloutput/USBRelay.h

```diff
@@ -1,42 +1,31 @@
-#pragma once
-/*
- * This file is part of the Falcon Player (FPP) and is Copyright (C)
- * 2013-2022 by the Falcon Player Developers.
- *
- * The Falcon Player (FPP) is free software, and is covered under
- * multiple Open Source licenses.  Please see the included 'LICENSES'
- * file for descriptions of what files are covered by each license.
- *
- * This source file is covered under the GPL v2 as described in the
- * included LICENSE.GPL file.
- */
-
-#include <string>
+#ifndef _USBRELAY_H
+#define _USBRELAY_H
 
 #include "ChannelOutput.h"
-#include "SerialChannelOutput.h"
+// Remove #include "SerialChannelOutput.h" since serialutil.h handles serial ops
```

Comment:
Please restore the use of SerialChannelOutput so that the capes can properly map things.